### PR TITLE
modified show-birthdays

### PR DESCRIPTION
**Pull request by Mz-Vl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -146,4 +146,7 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-cache
\ No newline at end of file
+cache
+
+#temp files
+tempCodeRunnerFile.py
```
Comment (kazamov) [inline .gitignore]: Revert the change in this file. If you want to have some file you just should not commit it

File: neoassistant/commands.py [hunk 2]
```diff
@@ -241,10 +241,22 @@ def execute(self, assistant: Assistant, args):
 
 class ShowBirthdaysCommand(Command):
     def __init__(self):
-        super().__init__("show-birthdays", "Show all birthdays per next 7 days.")
+        super().__init__(
+            "show-birthdays", 
+            "Show all birthdays per the next specified number of days," +
```
Comment (kazamov) [inline neoassistant/commands.py]: Update to `"Show all birthdays per the next specified number of days. Format: show-birthdays <days> (default 7)"`

File: neoassistant/commands.py [hunk 3]
```diff
@@ -241,10 +241,22 @@ def execute(self, assistant: Assistant, args):
...
+            "show-birthdays", 
+            "Show all birthdays per the next specified number of days," + 
+            " if days are not specified show per next 7 days."
+            )
 
-    def execute(self, assistant: Assistant, _):
-        return assistant.contact_book.get_birthdays_per_week()
+    def execute(self, assistant: Assistant, args):
+        days_delta = 7
+        if args:
```
Comment (kazamov) [inline neoassistant/commands.py]: Remove `if` check because it is always true, if there is not argument the list will be empty but it is a list, not a None

File: neoassistant/commands.py [hunk 4]
```diff
@@ -241,10 +241,22 @@ def execute(self, assistant: Assistant, args):
...
+            try:
+                days_delta = int(args[0])
+                if days_delta > 365:
+                    return "The maximum value for days_delta is 365."
```
Comment (kazamov) [inline neoassistant/commands.py]: Please raise an error instead of returning a string: `raise InvalidCommandError(self.name, "The maximum value for days_delta is 365.")`

File: neoassistant/commands.py [hunk 5]
```diff
@@ -241,10 +241,22 @@ def execute(self, assistant: Assistant, args):
...
+            except ValueError:
+                return "Invalid numbers of days"
```
Comment (kazamov) [inline neoassistant/commands.py]: Please raise an error instead of returning a string: `raise InvalidCommandError(self.name, "Invalid numbers of days.")`

**Review by kazamov — CHANGES_REQUESTED** [review]
Please fix the comments

File: neoassistant/commands.py (hunk 3 above)
Comment (Mz-Vl) [reply neoassistant/commands.py]: When I remove `if` check, the show-birthdays command without parameters raise an error.
`Enter a command: show-birthdays  
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "E:\Master Woolf\goitneo-python-final-project-group-11\.venv\Scripts\neoassistant.exe\__main__.py", line 7, in <module>     
  File "E:\Master Woolf\goitneo-python-final-project-group-11\neoassistant\__main__.py", line 27, in main
    print(f"\n{command_object.execute(neoassistant, args)}\n")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\Master Woolf\goitneo-python-final-project-group-11\neoassistant\commands.py", line 254, in execute
    days_delta = int(args[0])
                     ~~~~^^^
IndexError: list index out of range`

**Review by kazamov — APPROVED** [review]
